code/_core/obj/structure/interactive/nature/snow.dm

```diff
@@ -94,6 +94,8 @@ obj/structure/scenery/snow_bush/New()
 
 /obj/structure/interactive/tree/evergreen/New(var/desired_loc)
 	icon_state = "evergreen_[rand(1,3)]"
+	pixel_x = rand(-17,-15)
+	pixel_y = rand(0,2)
```

Comment:
this could probably be put on the /obj/structure/interactive/tree interactive structure instead.

Reply:
You'll find that different trees need different offsets, unless oyu can 'pixel +='?

Reply:
Cool, it does work. Yeah, I'll add it to /tree